packages/mnemonic/src/mnemonic.ts

```diff
@@ -130,6 +130,35 @@ class Mnemonic {
     return Mnemonic.masterKeysFromSeed(seed);
   }
 
+  static isValidMnemonic(phrase: string) {
```

Comment:
```suggestion
  static isMnemonicValid(phrase: string) {
```

Like @arboleya said, I would suggest renaming the function